### PR TITLE
Cart

### PR DESCRIPTION
**Pull request by starver20**
Added responsive page for cart management.
Added dummy index.html to sort netlify check errors.
Use netlify preview link below.

### CODE REVIEW

**Review by mohammed-sas — APPROVED** [review]
suggestion : it would be nice if spacing to the left is given to the "prod-name"
pr good to merge

**Comment by starver20** [comment]
> suggestion : it would be nice if spacing to the left is given to the "prod-name" pr good to merge

Thank you, Implemented the suggestion.

File: cart/cart.css [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+  font-size: 2rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 2px 4px 4px;
+}
+
+.remv-cart-btn {
+  /* border: 1px solid var(--sub-text-color); */
+  /* background-color: var(--background-color); */
```
Comment (kart-c) [inline cart/cart.css]: you can remove these lines if they are not required

**Review by kart-c — APPROVED** [review]
Cart page looks good Amar, just a suggestion you should remove the commented code in `cart.css` if it's not a part of the file. Also a pointer cursor would look good on the `+` and `-` buttons

File: cart/cart.css (hunk 1 above)
Comment (starver20) [reply cart/cart.css]: Thanks, ive implemented the changes

**Pull request merged**